src/main/java/mate/controller/LoginController.java

```diff
@@ -0,0 +1,46 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final String DRIVER_ID = "driver_id";
+    private static final String NAME = "name";
+    private static final String ERROR_VARIABLE = "error";
+    private static final String LOGIN = "login";
+    private static final String PASSWORD = "password";
```

Comment:
```suggestion
    private static final String DRIVER_ID = "driver_id";
```

I think we need to make as constant only "driver_id", because perhaps in the future we could change this parameter, but it is very unlikely that we will need to change "name", "login" and "password" and the fact that we make them constants will not increase the readability of the code.

Reply:
understand! thank you!

Reply:
"name" also can be change in future, can I to save it?